cmd/fleet/cron.go

```diff
@@ -814,6 +814,11 @@ func newCleanupsAndAggregationSchedule(
 				if err = ds.CleanupGlobalDiscardQueryResults(ctx); err != nil {
 					return err
 				}
+				return nil
+			}
+
+			if err := ds.CleanupQueryResultsWithDiscardData(ctx); err != nil {
```

Comment:
thinking this could take a while, maybe we should start logging execution times for each "tasks" within this cron